src/Package/Impl/DataInspect/Viewers/ObjectDetailsViewerProvider.cs

```diff
@@ -28,8 +29,8 @@ public sealed class ObjectDetailsViewerProvider : IObjectViewer {
             }
         }
 
-        public async Task ViewFile(string fileName, string tabName, bool deleteFile) {
-            await VsAppShell.Current.SwitchToMainThreadAsync();
+        public async Task ViewFile(string fileName, string tabName, bool deleteFile, CancellationToken cancellationToken) {
+            await VsAppShell.Current.SwitchToMainThreadAsync(cancellationToken);
```

Comment:
There are probably many other places that do it in non-cancellable way. Do we care?